debian/patches/no_local_serve_XXX

```diff
@@ -0,0 +1,13 @@
+diff --git a/datalad/tests/utils.py b/datalad/tests/utils.py
+index cc3ae14..6ebfc62 100644
+--- a/datalad/tests/utils.py
++++ b/datalad/tests/utils.py
+@@ -253,6 +253,8 @@ def _multiproc_serve_path_via_http(hostname, path_to_serve_from, queue): # pragm
+ def serve_path_via_http(tfunc, *targs):
+     """Decorator which serves content of a directory via http url
+     """
++    # XXX put back once I figured out why this thing doesn't work at package build time
++    raise SkipTest
```

Comment:
as I have commented (https://github.com/datalad/datalad/issues/212#issuecomment-140710790) most probable cause is pybuild defining/overiding http_proxy.  2.20150728 of pybuild (obviously only in recent debians) would allow to specify to not override it by setting it empty.  Unfortunately, with earlier versions of pybuild, such empty setting `export http_proxy=` wouldn't work for us either since haskell's libraries do not digest it well so git-annex can't operate either :-/    [...removed previous reply...]  PR is coming to fix it up!